src/index.ts

```diff
@@ -42,10 +42,11 @@ app.set('view engine', 'pug')
 app.use(morgan('combined'))
 app.use(express.static('offsets'))
 let connectionCount = 0;
+let publicConnectionCount = process.env.PUBLIC_CONNECTION_COUNT || connectionCount;
```

Comment:
This variable will never change. In JS, with strings and numbers, you're not storing a reference, you're storing a value. If you wrapped this in an object that would work.